galette/templates/default/pages/contributions_list.html.twig

```diff
@@ -242,72 +306,14 @@
 {% endblock %}
 
 {% block header %}
-    {% set columns = [
-        {
-            'label': '#',
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_ID"),
-            'collapse': true
-        },
-        {
-            'label': _T("Date"),
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_DATE"),
-            'collapse': true
-        },
-        {
-            'label': _T("Begin"),
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_BEGIN_DATE"),
-            'collapse': true
-        },
-        {
-            'label': _T("End"),
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_END_DATE"),
-            'collapse': true
-        }
-    ] %}
-
-    {% if ((login.isAdmin() or login.isStaff()) and member is not defined) or pmember is defined %}
-        {% set columns = columns|merge([{'label': _T("Member"), 'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_MEMBER")}]) %}
-    {% endif %}
-
-    {% set columns = columns|merge([
-        {
-            'label': _T('Type'),
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_TYPE")},
-        {
-            'label': _T('Amount'),
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_AMOUNT"),
-            'collapse': true
-        }
-    ]) %}
-
-    {% if preferences.pref_use_paid_flag %}
-        {% set columns = columns|merge([
-            {
-                'label': _Tx('contribution', 'Paid'),
-                'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_PAID"),
-                'collapse': true
-            }
-        ]) %}
-    {% endif %}
-
-    {% set columns = columns|merge([
-        {
-            'label': _T("Payment type"),
-            'order': constant("Galette\\Filters\\ContributionsList::ORDERBY_PAYMENT_TYPE")},
-        {
-            'label': _T("Duration"),
-            'collapse': true
-        }
-    ]) %}
-
     {{ parent() }}
 {% endblock %}
 
 {% block footer %}
     {% if mode != 'ajax' and nb != 0 %}
         <tr>
-            <th class="right aligned" colspan="{{ preferences.pref_use_paid_flag ? '11' : '10' }}">
-                {{ _T("Found contributions total %f")|replace({"%f": contribs.getSum()}) }}
+            <th class="right aligned" colspan="{{ columns_count }}" scope="colgroup">
+                {{ _T("Total amount of contributions: %f")|replace({"%f": contribs.getSum()}) }}
```

Comment:
I'm not sure of the sentence change. I've used "found" to make things clear: this is not the total amount of contributions; but the total of found ones with current filters.